http-kernel-fixtures/advanced_form_post.php

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
 </head>
 <body>
+<pre>
```

Comment:
Adding a ``<pre>`` tag is not necessary for the refactoring here (I did a different attempt before where it was useful). But it makes the page more readable when looking at it, as our dumping relies on preformatted text. The testsuite itself does not care, as it asserts on the HTML source, not on the visual rendering.